### DISCUSSION TITLE
Mount other server

### DISCUSSION THREAD

**Discussion by masterwotikar** [drakkan/sftpgo#883] [Q&A]
I have a mount point on my docker host to another SMB server. I want to serve files through webdav with sftpgo. Currently i see the files of the moint in my webdav client, but sftpgo logs show invalid root path.

**Comment by drakkan** [comment]
Please post the steps to reproduce your configuration and some SFTPGo logs. The root dir must be a valid absolute path. This could be the problem or maybe you have a symlink that points outside the defined root dir
